src/viewer.js

```diff
@@ -2983,7 +2992,13 @@ function onCanvasDrag( event ) {
 
     gestureSettings = this.gestureSettingsByDeviceType( event.pointerType );
 
-    if ( gestureSettings.dragToPan && !canvasDragEventArgs.preventDefaultAction && this.viewport ) {
+    if (gestureSettings.dblClickToZoom && THIS[ this.hash ].draggingToZoom){
```

Comment:
This statement needs `!canvasDragEventArgs.preventDefaultAction && this.viewport` just like the other one. Probably would be good to wrap both of these if statements in one that has that logic.